crates/cheatnet/tests/contracts/src/bytearray_string_panic_call.cairo

```diff
@@ -0,0 +1,46 @@
+use starknet::ContractAddress;
+
+#[starknet::interface]
+trait IByteArrayPanickingContract<TContractState> {
+    fn do_panic(self: @TContractState);
+}
+
+// ByteArrayPanickingContract
+#[starknet::contract]
+mod BAPC {
```

Comment:
Can't we change this abbreviation to the full name?

Reply:
bapc 👍

Reply:
No, it doesn't fit into felt :P

Reply:
This one should fit. It has 26 characters

Reply:
done